Source/CesiumRuntime/Private/CesiumTextureUtility.cpp

```diff
@@ -372,7 +407,7 @@ static UTexture2D* CreateTexture2D(LoadedTextureResult* pHalfLoadedTexture) {
 
     pTexture->NeverStream = true;
 
-    pHalfLoadedTexture->pTexture = new ReferenceCountedUnrealTexture(pTexture);
+    pHalfLoadedTexture->pTexture->setUnrealTexture(pTexture);
```

Comment:
Drive by comment... in theory we can get to this line when `pHalfLoadedTexture->pTexture` is equal to `nullptr` which line 390 implies is an expected case. 

If that happens, line 410 will deference a nullptr

Reply:
Good eye @csciguy8! `pTexture` can't be nullptr here, because it's always assigned immediately after creating the `LoadedTextureResult` (either by newing up an instance or by assigning an existing instance from the extension). I left the defensive check in, but moved it up so that there wouldn't be a null pointer dereference if it ever actually happened.